esphome/components/esphome/ota/ota_esphome.h

```diff
@@ -2,48 +2,19 @@
 
 #include "esphome/components/socket/socket.h"
 #include "esphome/core/component.h"
-#include "esphome/core/preferences.h"
-#include "esphome/core/helpers.h"
 #include "esphome/core/defines.h"
+#include "esphome/core/helpers.h"
+#include "esphome/core/preferences.h"
 
 namespace esphome {
-namespace ota {
-
-enum OTAResponseTypes {
-  OTA_RESPONSE_OK = 0x00,
-  OTA_RESPONSE_REQUEST_AUTH = 0x01,
-
-  OTA_RESPONSE_HEADER_OK = 0x40,
-  OTA_RESPONSE_AUTH_OK = 0x41,
-  OTA_RESPONSE_UPDATE_PREPARE_OK = 0x42,
-  OTA_RESPONSE_BIN_MD5_OK = 0x43,
-  OTA_RESPONSE_RECEIVE_OK = 0x44,
-  OTA_RESPONSE_UPDATE_END_OK = 0x45,
-  OTA_RESPONSE_SUPPORTS_COMPRESSION = 0x46,
-  OTA_RESPONSE_CHUNK_OK = 0x47,
-
-  OTA_RESPONSE_ERROR_MAGIC = 0x80,
-  OTA_RESPONSE_ERROR_UPDATE_PREPARE = 0x81,
-  OTA_RESPONSE_ERROR_AUTH_INVALID = 0x82,
-  OTA_RESPONSE_ERROR_WRITING_FLASH = 0x83,
-  OTA_RESPONSE_ERROR_UPDATE_END = 0x84,
-  OTA_RESPONSE_ERROR_INVALID_BOOTSTRAPPING = 0x85,
-  OTA_RESPONSE_ERROR_WRONG_CURRENT_FLASH_CONFIG = 0x86,
-  OTA_RESPONSE_ERROR_WRONG_NEW_FLASH_CONFIG = 0x87,
-  OTA_RESPONSE_ERROR_ESP8266_NOT_ENOUGH_SPACE = 0x88,
-  OTA_RESPONSE_ERROR_ESP32_NOT_ENOUGH_SPACE = 0x89,
-  OTA_RESPONSE_ERROR_NO_UPDATE_PARTITION = 0x8A,
-  OTA_RESPONSE_ERROR_MD5_MISMATCH = 0x8B,
-  OTA_RESPONSE_ERROR_RP2040_NOT_ENOUGH_SPACE = 0x8C,
-  OTA_RESPONSE_ERROR_UNKNOWN = 0xFF,
-};
+namespace ota_esphome {
```

Comment:
Namespaces need to match the platform
```suggestion
namespace esphome {
```